shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -567,6 +567,9 @@ - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configur
   _view.keyboardType = ToUIKeyboardType(configuration[@"inputType"]);
   _view.autocapitalizationType = ToUITextAutocapitalizationType(configuration[@"inputType"]);
   _view.secureTextEntry = [configuration[@"obscureText"] boolValue];
+  _view.autocorrectionType = [configuration[@"autocorrect"] boolValue]
```

Comment:
Just saw that patch. The value cannot be nil. So I think it is safe to not have a default in there.